### PR TITLE
{bio}[foss/2018b] MuPeXI v1.2.0, jemalloc v5.1.0, MariaDB v10.3.10, ... (REVIEW)

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/d/DBD-mysql/DBD-mysql-4.048_mariadb_base_version.patch [hunk 1]
```diff
@@ -0,0 +1,20 @@
+diff -ur DBD-mysql-4.048.orig/dbdimp.c DBD-mysql-4.048/dbdimp.c
```
Comment (boegel) [inline easybuild/easyconfigs/d/DBD-mysql/DBD-mysql-4.048_mariadb_base_version.patch]: @smoors Please include a short description of the patch + mention author or upstream reference to source of the patch (and don't forget to update the checksum accordingly)

File: easybuild/easyconfigs/m/MuPeXI/MuPeXI-1.2.0-foss-2018b-Perl-5.28.0-Python-2.7.15.eb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+postinstallcmds = [
+    'sed -i -e "s|^MHC =.*|MHC = $EBROOTNETMHCPAN/netMHCpan|" %(installdir)s/config.ini',
+    'sed -i -e "s|^VEP =.*|VEP = $EBROOTVEP/vep|" %(installdir)s/config.ini',
+    'sed -i -e "s|^PM =.*|PM = %(installdir)s/bin/pepmatch_db_x86_64|" %(installdir)s/config.ini',
+]
+
+modextrapaths = {'PATH': ''}
+
+sanity_check_paths = {
+    'files': ['config.ini', 'MuPeXI.py'],
```
Comment (boegel) [inline easybuild/easyconfigs/m/MuPeXI/MuPeXI-1.2.0-foss-2018b-Perl-5.28.0-Python-2.7.15.eb]: @smoors Also check for `bin/pepmatch_db_x86_64`?

File: easybuild/easyconfigs/m/MuPeXI/MuPeXI-1.2.0-foss-2018b-Perl-5.28.0-Python-2.7.15.eb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    'dirs': [],
+}
+
+sanity_check_commands = [('MuPeXI.py -h')]
```
Comment (boegel) [inline easybuild/easyconfigs/m/MuPeXI/MuPeXI-1.2.0-foss-2018b-Perl-5.28.0-Python-2.7.15.eb]: @smoors No need for the `(...)`:

```python
sanity_check_commands = ["MuPeXI.py -h"]

File: easybuild/easyconfigs/n/netMHCpan/netMHCpan-4.0a.eb [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+ molecule using artificial neural networks (ANNs). """
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+# netMHCpan is proprietary software, but free for academcs
+# see: http://www.cbs.dtu.dk/cgi-bin/nph-sw_request?netMHCpan
+source_urls = ['http://www.cbs.dtu.dk/services/NetMHCpan-4.0']
+sources = [
+    '%(name)s-%(version)s.Linux.tar.gz',
+    {'filename': 'data.Linux.tar.gz', 'extract_cmd': "tar -xzf %s -C %(builddir)s/%(name)s-4.0/Linux_x86_64"},
```
Comment (boegel) [inline easybuild/easyconfigs/n/netMHCpan/netMHCpan-4.0a.eb]: @smoors Shouldn't we rename this file on download using `%(version)s`?

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 7 out of 9 (9 easyconfigs in this PR)
node2585.golett.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/6c9fa471bbaa14f6d38a562a6628a726 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 7 out of 9 (9 easyconfigs in this PR)
node2060.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1b9c684450bca2a9b1fa665dc00b8543 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 8 out of 10 (9 easyconfigs in this PR)
node3111.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/c14fab6e78ffec9f8821b875f246cc39 for a full test report.

**Comment by smoors** [comment]
@boegel `netMHCpan` sources must be downloaded separately (you can get it here: /apps/brussel/sources)

**Comment by smoors** [comment]
please do not merge yet, I need to make a small fix.

**Comment by smoors** [comment]
@boegel cfr. changing file permissions in the `netMHCpan` easyconfig: shouldn't this be done by default in the `Tarball` easyblock?

**Comment by smoors** [comment]
@boegel my question is: does it have to know? why not always do `chmod -R +rX` after unpacking the tarball?

**Comment by smoors** [comment]
this PR has 10 easyconfigs, can this be merged soon?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
node3112.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/2763468a69ccadc370faf44fd8268479 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
node2715.swalot.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ff88873cb0e61dfc7b69fd74d2f29a03 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in this PR)
node2089.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/903a65068d0e36d9ea17ef11957de878 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @smoors!

**Pull request merged**